source/javascripts/controllers/_StackController.js.erb

```diff
@@ -163,9 +163,16 @@
 				if (stack.descriptionURLGen2 !== undefined) {
 					return stack.descriptionURLGen2;
 				}
-				return stack.descriptionURL;
+
+				if (stack.descriptionURL) {
+					return stack.descriptionURL;
+				}
```

Comment:
If `stack.descriptionURL` is falsy (null, undefined, "") then the `viewModel.descriptionURLForStack` should be the same too. However, this solution always returns undefined, I think we shouldn't want that.